python/amici/sbml_import.py

```diff
@@ -295,6 +295,7 @@ def sbml2amici(self,
                    sigmas=None,
                    verbose=False,
                    assume_pow_positivity=False,
+                   compiler=None
```

Comment:
required for windows ('mingw32'), just wonder if amici can memorize this at compile time

Reply:
Could be convenient. Would have to create a config file in package directory. Feel free to open a separate issue on that.